contact.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Artūras Adomaitis</title>
+    <link rel="stylesheet" href="stilius.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.2/css/all.min.css">
+</head>
+<body>
+    <div class="page-top">
+        <br>
+        <h1>Artūras Adomaitis</h1>
+        <p id="subheading">Python Developer</p>
+    </div>
+    <nav>
+        <ul class="nav-list">
+            <li class="nav-item"><a href="index.html"><strong>Home</strong></a></li>
+            <li class="nav-item"><a href="about.html"><strong>About</strong></a></li>
+            <li class="nav-item"><a href="#"><strong>Contact</strong></a></li>
+        </ul>
+    </nav>
+    <p id="tape"></p>
+    <div class="contact-form">
+        <p id="write-me"><strong>Don't hesitate to contact me. Please fill the form below and your message will be send.</strong></p>
+        <form>
+            <input type="email" name = "email" size="46" placeholder="Your Email address"><br><br>
+            <input type="text" name = "subject" size="46" placeholder="Subject"><br><br>
+            <input type="text" class="wideform" name = "formtext" size="46" placeholder="Your message"><br><br><br>
+            <button type="submit">Send</button>
+        </form><br><br>
+    </div>
+    <div id="bottom-elements">
+        <a href="https://www.linkedin.com/in/art%C5%ABras-adomaitis-33742612/" class="fa-brands fa-linkedin-in" id="linkedin"></a>
+        <a href="mailto: arturas.adomaitis@gmail.com" class="fa-regular fa-envelope" id="email"></a>
+    </div>
```

Comment:
Footeris kartojasi visuose puslapiuose. Gal reikėjo tiesiog daryt vieną html failą ir priklausomai nuo to, kokį puslapį pasirenka naudotojus kai kuriuos puslapio elementus tiesiog paslėpti?